framework/Optimizers/GradientBasedOptimizer.py

```diff
@@ -605,3 +646,13 @@ def getQueuedPoint(self,traj,denorm=True):
       point = self.denormalizeData(point)
     return prefix,point
 
+  def proposeNewPoint(self,traj,point):
+    """
+      See base class.  Used to set next recommended point to use for algorithm, overriding the gradient descent.
+      @ In, traj, int, trajectory who gets proposed point
+      @ In, point, dict, input space as dictionary {var:val}
+      @ Out, None
+    """
+    Optimizer.proposeNewPoint(self,traj,point)
```

Comment:
The "propose new point" is currently only used in Preconditioning, where instead of evaluating the gradient to determine the next point, we simply accept the preconditioner's suggestion as a theoretical new point and evaluate the model at the preconditioner-chosen point.  

There isn't any gradient evaluation being performed in this case, as can be seen in the TargetEvaluation output, where the runs with even prefixes are missing on preconditioned points.